dpc/src/parameters/testnet1.rs

```diff
@@ -149,9 +148,13 @@ impl Parameters for Testnet1Parameters {
     type ProgramCommitmentGadget = Blake2sCommitmentGadget;
     type ProgramCommitment = <Self::ProgramCommitmentScheme as CommitmentScheme>::Output;
 
-    type ProgramCircuitIDCRH = PoseidonCryptoHash<Self::OuterScalarField, 4, false>;
-    type ProgramCircuitIDCRHGadget = PoseidonCryptoHashGadget<Self::OuterScalarField, 4, false>;
+    type ProgramCircuitIDCRH = BHPCompressedCRH<EdwardsBW6, 237, 16>;
+    type ProgramCircuitIDCRHGadget = BHPCompressedCRHGadget<EdwardsBW6, Self::OuterScalarField, EdwardsBW6Gadget, 237, 16>;
     type ProgramCircuitID = <Self::ProgramCircuitIDCRH as CRH>::Output;
+
+    type ProgramCircuitIDTreeCRH = BHPCompressedCRH<EdwardsBW6, 8, 48>;
+    type ProgramCircuitIDTreeCRHGadget = BHPCompressedCRHGadget<EdwardsBW6, Self::OuterScalarField, EdwardsBW6Gadget, 8, 48>;
+    type ProgramCircuitIDTreeDigest = <Self::ProgramCircuitIDTreeCRH as CRH>::Output;
     type ProgramCircuitTreeParameters = ProgramIDMerkleTreeParameters;
```

Comment:
We separate the `ProgramCircuitIDCRH` and  `ProgramCircuitIDTreeCRH` because the former is too large, and due to a reason about the current BHP implementation, it has an additional cost. So, the tree would invoke BHP of a smaller window. This is the same as our treatment for record commitment trees.